common/chain.go

```diff
@@ -39,15 +43,45 @@ func (chain Chain) IsExternalChain() bool {
 	return !chain.IsEqual(ZetaChain())
 }
 
+// bytes representations of address
+// on EVM chain, it is 20Bytes
+// on Bitcoin chain, it is P2WPKH address, []byte(bech32 encoded string)
+func (chain Chain) EncodeAddress(b []byte) (string, error) {
+	if IsEVMChain(chain.ChainId) {
+		addr := ethcommon.BytesToAddress(b)
+		if addr == (ethcommon.Address{}) {
+			return "", fmt.Errorf("invalid EVM address")
+		}
+		return addr.Hex(), nil
+	} else if IsBitcoinChain(chain.ChainId) {
+		addrStr := string(b)
+		var chainParams *chaincfg.Params
+		switch chain.ChainId {
+		case 18444:
```

Comment:
Can we modify the chain name in default_chains_XXX to the some name for BTC, so that w can avoid this switch case .